meteo-notebook.ipynb

```diff
@@ -110,7 +110,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    one_week = df_meteo[df_meteo['date']. dt. strftime('%Y-%m')=='2021-11'].head(168)
</pre><p>Which postalcode are you showing exactly?</p><p> </p><p>Where did you get 168 from?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rawan-a-d/Air-Quality-Prediction/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>